bot.py

```diff
@@ -5,26 +5,69 @@
 
 client = commands.Bot(command_prefix = "$")

```

Comment:
You should abstract out the `command_prefix` so you can use it outside of this context.

In your help messages, you hard-code the `$` - instead, use a format string with this `command_prefix`.